tests/utils/test_simulator.py

```diff
@@ -0,0 +1,154 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Brain Simulator test script
+
+Test script for generating a run of a participant's data.
+
+ Authors: Cameron Ellis (Princeton) 2016
+"""
+import numpy as np
+import math
+from brainiak.utils import simulator as sim
+
+
+# Inputs for generate_signal
+dimensions = np.array([64, 64, 36]) # What is the size of the brain
```

Comment:
Testing modules should be composed of `test_`-prefixed functions (and, if necessary, `Test`-prefixed classes):
http://docs.pytest.org/en/latest/goodpractices.html#test-discovery